src/auth/screens/language-settings.js

```diff
@@ -29,4 +29,9 @@ export default [
     emojiCode: ':flag-ru:',
     name: 'Русский',
   },
+  {
+    code: 'de',
+    emojiCode: ':flag-de:',
+    name: 'Deutsche',
```

Comment:
Why not `Deutsch`?

Reply:
as i was shown this result [here](https://translate.google.com/#en/de/german)
Should i change it to ``Deutsch``?

Reply:
Yes, because we use the adjective.

`Deutsch` - adjective
`Deutsche` - noun